deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,50 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body {
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+/*////////////////////Navigatie////////////////////////////*/
+
+.col-1 {
+    flex-basis: 250px;
+    flex-grow: 0;
+    flex-shrink: 0;
+}
+
+/*//////////////////Basis/////////////////////////////*/
+
+.col-2 {
+    border: 3px dashed blue;
+    flex-basis: 70vw;
```

Comment:
flex-basis had je beter op auto kunnen zetten . "Het element mag wél groter worden dan de basis afmeting als er meer ruimte is "